deel/torchlip/modules/pooling.py

```diff
@@ -204,3 +204,56 @@ def forward(self, input: torch.Tensor) -> torch.Tensor:
 
     def vanilla_export(self):
         return self
+
+
+class ScaledGlobalL2NormPool2d(torch.nn.AdaptiveAvgPool2d, LipschitzModule):
```

Comment:
torch LPPool2D internally uses `torch.nn.functional.lp_pool2d`, shouldn't we use the same function in this layer?